geoserver/geoserver_seeder/geoserver_seeder.rb

```diff
@@ -8,344 +8,36 @@
 require 'open-uri'
 require 'logger'
 
+require_relative 'layer_seeder'
+require_relative 'geonetwork_connector'
+require_relative 'geowebcache'
+
 $logger = Logger.new(STDOUT)
 $logger.level = Logger::INFO
 
-class TileGenerator
```

Comment:
I agree. We can't go straight to using production portal with geowebcache. We'll need to keep the old seeder and caching for at least an iteration while we allow testing to occur.